python_modules/dagster/dagster/core/execution/context/system.py

```diff
@@ -457,6 +457,11 @@ def can_load(self, step_output_handle: StepOutputHandle) -> bool:
             if step_output_handle == record.dagster_event.event_specific_data.step_output_handle:
                 return True
 
+        # source is skipped so cannot load
```

Comment:
This doesn't solve the entire problem we talked abut yesterday, right? Because it's possible that the step itself didn't skip, but rather that it chose not to yield the output in question?